Makefile

```diff
@@ -54,6 +54,16 @@ release: bins
 test: vendor
 	CGO_ENABLED=0 go test -v ./...
 
+testacc: vendor
+	TF_ACC=1 CGO_ENABLED=0 go test -v --cover ./... -timeout 120m
+
+sweep:
+	@echo "WARNING: This will destroy infrastructure. Use only in development accounts."
+	go test -v ./... -sweep=global -timeout 60m
+
+fmt: vendor
+	! go fmt ./... 2>&1 | read
```

Comment:
Is putting this target back intentional? It was just removed in an earlier commit

Reply:
a good catch, it shouldn't be there